site/content/en/docs/Guides/feature-stages.md

```diff
@@ -27,37 +27,41 @@ be disabled.
+| [Rolling Update Fixes](https://github.com/googleforgames/agones/issues/3688)                                               | `RollingUpdateFix`                 | Disabled | `Alpha` | 1.41.0 |
+| [Multiple dynamic port ranges](https://github.com/googleforgames/agones/issues/1911)                                       | `PortRanges`                       | Disabled | `Alpha` | 1.41.0 |
+| [Port Policy None](https://github.com/googleforgames/agones/issues/3804)                                                   | `PortPolicyNone`                   | Disabled | `Alpha` | 1.41.0 |
+| [Scheduled Fleet Autoscaling](https://github.com/googleforgames/agones/issues/3008)                                        | `ScheduledAutoscaler`              | Disabled | `Alpha` | 1.43.0 |
+| [Extend Webhook autoscaler to send fleet metadata with the request](https://github.com/googleforgames/agones/issues/3951)  | `FleetAutoscaleRequestMetaData`    | Disabled | `Alpha` | 1.48.0 |
+| Example Gate (not in use)                                                                                                  | `Example`                          | Disabled | None    | 0.13.0 |
+
 {{% /feature %}}
 
 {{% feature publishVersion="1.49.0" %}}
+
 | Feature Name                                                                                                               | Gate                               | Default  | Stage    | Since  |
 |----------------------------------------------------------------------------------------------------------------------------|------------------------------------|----------|----------|--------|
 | [CountsAndLists](https://github.com/googleforgames/agones/issues/2716)                                                     | `CountsAndLists`                   | Enabled  | `Beta`   | 1.41.0 |
 | [DisableResyncOnSDKServer](https://github.com/googleforgames/agones/issues/3377)                                           | `DisableResyncOnSDKServer`         | Enabled  | `Stable` | 1.49.0 |
 | [Support Passthrough on GKE Autopilot](https://github.com/googleforgames/agones/issues/3721)                               | `AutopilotPassthroughPort`         | Enabled  | `Beta`   | 1.43.0 |
-| [Support for Extended Duration Pods on GKE Autopilot (*1.28+ only*)](https://github.com/googleforgames/agones/issues/3386) | `GKEAutopilotExtendedDurationPods` | Enabled  | `Beta`   | 1.44.0 |
-| [Port Policy None](https://github.com/googleforgames/agones/issues/3804)                                                   | `PortPolicyNone`                   | Enabled  | `Beta`   | 1.49.0 |
+| [Support for Extended Duration Pods on GKE Autopilot (*1.28+ only*)](https://github.com/googleforgames/agones/issues/3386) | `GKEAutopilotExtendedDurationPods` | Enabled  | `Beta`   | 1.44.0 || [Port Policy None](https://github.com/googleforgames/agones/issues/3804)                                                   | `PortPolicyNone`                   | Enabled  | `Beta`   | 1.49.0 |
```

Comment:
There seems to be a missing new line here, might not show in the table
```suggestion
| [Support for Extended Duration Pods on GKE Autopilot (*1.28+ only*)](https://github.com/googleforgames/agones/issues/3386) | `GKEAutopilotExtendedDurationPods` | Enabled  | `Beta`   | 1.44.0 |
| [Port Policy None](https://github.com/googleforgames/agones/issues/3804)                                                   | `PortPolicyNone`                   | Enabled  | `Beta`   | 1.49.0 |
```

Reply:
Well spotted!